hbw/app/javascript/packs/hbw/components/form/file_upload.js.jsx

```diff
@@ -80,11 +81,12 @@ modulejs.define('HBWFormFileUpload', ['React'], (React) => {
       } = this.props;
 
       const { isDragActive } = this.state;
+      const fileInputID = uuidv4();
```

Comment:
I think there is no need to generate it on every render, make it a component property.